src/App.tsx

```diff
@@ -1,24 +1,148 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  FC, useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { ErrorType } from './types/Error';
+import { TaskStatus } from './types/Sort';
+import { deleteTodo, getTodos, postTodo } from './api/todos';
+import { AddTodo } from './components/AddTodo';
+import { TodoList } from './components/TodoList';
+import { Notification } from './components/Notification';
+import { Filter } from './components/Filter';
+import { closeErrorMessage, getFilteredTodos } from './utils/helpers';
+import { DEFAULT_TASK_ID, USER_ID } from './utils/constants';
 
-const USER_ID = 0;
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<ErrorType>(ErrorType.NONE);
+  const [sortType, setSortType] = useState<TaskStatus>(TaskStatus.ALL);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isDisabled, setIsDisabled] = useState<boolean>(false);
+  const [loadTodoById, setLoadTodoById] = useState([DEFAULT_TASK_ID]);
+
+  const fetchTodos = async () => {
+    try {
+      const getData = await getTodos(USER_ID);
+
+      setTodos(getData);
+    } catch {
+      setError(ErrorType.LOAD);
+      closeErrorMessage(ErrorType.NONE);
+    }
+  };
```

Comment:
Better to use useCallback with such methods. Please fix it here and down below in your code